packages/modelviewer.dev/examples/postprocessing/index.html

```diff
@@ -0,0 +1,364 @@
+            <example-snippet>
+              <template>
+<script type="module-no-exec" async src="https://ga.jspm.io/npm:es-module-shims@1.6.3/dist/es-module-shims.js"></script>
+              </template>
+            </example-snippet>
+            <p>Bring <code>Three.js</code> from any cdn - cdnjs, unpkg, or jsdelivr:</p>
+            <example-snippet>
+              <template>
+<script type="importmap-no-exec">
+{
+  "imports": {
+    "three": "https://cdn.jsdelivr.net/npm/three@0.150.1/build/three.module.js"
+  }
+}
+</script>
+              </template>
+            </example-snippet>
+            <p>Now import the &lt;model-viewer&gt; module version which doesn't package three, and the &lt;model-viewer-effects&gt; addon</p>
+            <example-snippet stamp-to="setup" highlight-as="html">
+              <template>
+<script type="module-no-exec" src="https://cdn.jsdelivr.net/npm/@google/model-viewer/dist/model-viewer-module.min.js"></script>
+  
+<script type="module-no-exec" src="https://cdn.jsdelivr.net/npm/@google/model-viewer-effects/dist/model-viewer-effects.min.js"></script>
+              </template>
+            </example-snippet>
+            <p>Setting up post-processing effects is as simple as adding an &lt;effect-composer&gt; component inside your &lt;model-viewer&gt; element.
+              From here, simply add any of the supported effects inside the &lt;effect-composer&gt;.</p>
+          </div>
+          <example-snippet stamp-to="setup" highlight-as="html">
+            <template>
+<!-- Use the <effect-composer> component inside the <model-viewer> element -->
+<model-viewer autoplay ar camera-controls touch-action="pan-y" src="../../shared-assets/models/Horse.glb" alt="A 3D model of a horse galloping.">
+  <effect-composer>
+    <pixelate-effect></pixelate-effect>
+  </effect-composer>
+</model-viewer>
+            </template>
+          </example-snippet>
+        </div>
+      </div>
+    </div>
+
+
+    <div class="sample">
+      <div id="effects" class="demo"></div>
+      <div class="content">
+        <div class="wrapper">
+          <div class="heading">
+            <h2 class="demo-title">The Effects API</h2>
+            <p>All Effects are reactive components, so all properties can be changed during runtime. 
+            This example demonstrates the BlendMode API, which allows you to blend effects like you would
+            layers in Photoshop. We use a &lt;color-grade-effect&gt;, which allows you to alter the look of your model.
+            </p>
+          </div>
+          <example-snippet stamp-to="effects" highlight-as="html">
+            <template>
+<model-viewer id="blendViewer" camera-controls touch-action="pan-y" ar src="../../shared-assets/models/Astronaut.glb" alt="A 3D model of an astronaut">
+  <effect-composer>
+    <color-grade-effect id="color-grade" contrast="0.5" saturation="0.4"></color-grade-effect>
```

Comment:
Why is there heavy aliasing and strange transparency effects here? 
<img width="609" alt="image" src="https://user-images.githubusercontent.com/1649964/229639648-a1d4e230-8ca3-4dec-980d-aad29822f2e8.png">

Reply:
Interesting, it works correctly for me on Firefox, but has this same problem on both Chrome and Safari (I'm on an M1 Macbook Pro). I can see the artifact on the Robot example too, around the eyes. It looks like a very imprecise depth buffer to me, which is often caused by near and far planes that are much too far apart. Not sure where the browser-specific part comes in though...

Reply:
I've set the <code>render-mode='quality'</code> which uses a higher precision HalfFloatType buffers, this maybe might fix those issues for you?

Reply:
Not getting this on my macbook air 2013 (tested on chrome and firefox)

Reply:
Looks like this completely fixed it for me, thanks!